queue_job/migrations/13.0.1.0.0/post-migration.py

```diff
@@ -0,0 +1,10 @@
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from openupgradelib import openupgrade  # pylint: disable=W7936
+
+
+@openupgrade.migrate()
```

Comment:
Why do you need openupgrade? It should work OOTB in odoo and you could simply write into the rule.
In any case it should address 13.0.1.0.1 as patch version, no?

Reply:
This script is not meant to patch the running code. Is meant to handle the migration of the module from v12 to v13 and for this reason it's correct to run it when processing 13.0.1.0.0. When migrating from v12 to v13 the official OCA way is by openupgrade, or am I missing something?

Reply:
Yes, that's it. Any way, I recommend openupgradelib in any possible migration, being major or minor.

Reply:
@pedrobaeza for minor migrations I would not recommend openupgradelib since scripts will raise errors in platforms where openupgradelib is not installed, like odoo.sh. See for example https://github.com/OCA/hr/issues/754

Reply:
I see, but my point is: if you don't use openupgrade is going to work anyway. 
You just need a "migrations" folder w/ proper version number.
In this specific case, I don't see the benefit of using it.
I'm not against it, I'm just trying to understand :wink: